src/registries/npm-registry.js

```diff
@@ -177,7 +180,7 @@ export default class NpmRegistry extends Registry {
       }
     }
 
-    return '';
+    return undefined;
```

Comment:
Make this return an empty string please, we should ensure types are monomorphic if we can. An empty string will still evaluate the branch added above.